Farmland.py

```diff
@@ -1,205 +0,0 @@
-
-        if self.vertexList[0] < 0 or self.vertexList[1] < 0 or self.vertexList[2] >= fieldwidth or self.vertexList[3] >=fieldheight:
-            raise ValueError("The input of the vertex is out of bounds!")
-
-        #saves the vertex locations as attributes
-        self.lowerLeftX = self.vertexList[0]
-        self.lowerLeftY = self.vertexList[1]
-        self.upperRightX = self.vertexList[2]
-        self.upperRightY = self.vertexList[3]
-
-        #new attributes that describe the shape of the rectangle
-        self.height = 1+self.upperRightY - self.lowerLeftY
-        self.width = 1+self.upperRightX- self.lowerLeftX
-        self.area = self.height * self.width
-
-
-    #returns the individual squares of the barren land as points
-    def generatePoints(self):
-        Points = []
-
-        for i in range(0, self.width):
-            for j in range(0, self.height):
-                Points.append( [self.lowerLeftX + i, self.lowerLeftY + j])
-
-        return Points
-
-    #Prints BarrenLand coordinates
-    def getVertices(self):
-        LowerLeftVertex = [self.lowerLeftX, self.lowerLeftY]
-        UpperRightVertex = [self.upperRightX, self.upperRightY]
-
-        print("The lower left vertex is ", LowerLeftVertex )
-        print("The upper right vertex is ", UpperRightVertex)
-        return
-
-
-#This class is for the fertile land
-class FertileLand:
-
-    def __init__(self,fieldwidth= 400,fieldheight = 600):
-
-        #FertilePoints tells us if each 1 by 1 square is fertile or barren
-        #one represents fertile, zero represents barren
-        self.fertilePoints = np.matlib.ones((fieldwidth,fieldheight))
-        self.width = fieldwidth
-        self.height = fieldheight
-        self.totalArea = self.width * self.height #Holds the area of all fertile land
-
-        #seperated plots of land and number, we don't initialize this until removeBarren
-        self.plots_of_Land =[]  #0 if barren, >0 for the number of plot
-        self.number_of_Plots = 0 #counts the number of plots
-
-    #Prints FertilePoints
-    def printLand(self):
-        for i in range(self.height-1,0,-1):
-            for j in range (0,self.width):
-                print(self.fertilePoints[j,i], " ", end='')
-            print()
-        return
-
-
-    #changes FertilePoints to include barren land from a list of BarrenLand objects
-    def removeBarren(self, BarrenLandList):
-        for BarrenLand in BarrenLandList:
-            Barrenpoints = BarrenLand.generatePoints()
-            for point in Barrenpoints:
-                if self.fertilePoints[point[0],point[1]]==1:
-                    self.fertilePoints[point[0],point[1]]=0 #zero represents barren
-                    self.totalArea = self.totalArea - 1
-
-        return
-
-    #This should only be called after seperatePlots, it counts the area of each plot
-    def Area_of_Plots(self):

```

Comment:
You changed the name style of the function. This should be camelCase - areaOfPlots

Reply:
When you say name style do you mean using underscores? I realize that I don't use underscores for other functions